mediated_transfer.rst

```diff
@@ -0,0 +1,80 @@
+Mediated transfers
+##################
+
+Overview
+========
+
+Nodes can use a :term:`mediated transfer` to send payments to another node without opening a
+direct channel to it.
+
+A mediated transfer is done in two stages, usually through multiple channels:
+
+- **Allocation:** Reserve token :term:`capacity` for a given payment, using a
+  :ref:`locked transfer message <locked-transfer-message>`.
+- **Finalization:** Use the reserved token amount to complete payments, using the
+  :ref:`unlock message <unlock-message>`
+
+A mediated transfer may be cancelled and can expire until the initiator reveals the secret.
+
+Mediated Transfers
+==================
+
+A :term:`mediated transfer` is a hash-time-locked transfer. Currently raiden supports only one type of lock. The lock has an amount that is being transferred, a :term:`secrethash` used to verify the secret that unlocks it, and a :term:`lock expiration` to determine its validity.
+
+Mediated transfers have an :term:`initiator` and a :term:`target` and a number of mediators in between. The number of mediators can also be zero as these transfers can also be sent to a direct partner. Assuming ``N`` number of mediators, a mediated transfer will require ``10N + 16`` messages to complete. These are:
```

Comment:
- Can we somehow say that this "The number of mediators can also be zero as these transfers can also be sent to a direct partner." is an edge case and put it to the end of the description
- is it true that in this edge case, this ``10N + 16`` would mean that ther are 16 messages needed to complete a mediated transfers between two nodes without an mediator?

Reply:
I singled it out as a note. As far as I can tell there are indeed 16 messages in that case.